web/book/src/queries/pipelines.md

```diff
@@ -43,5 +43,5 @@ select {e.name, gross_salary, d.name}
 
 ## See also
 
-- [Transforms](../transforms/README.md) - PRQL Transforms
-- [Syntax](../syntax/README.md) - Notation for PRQL queries
+- [Transforms](../transforms/index.html) - PRQL Transforms
+- [Syntax](../syntax/index.html) - Notation for PRQL queries
```

Comment:
Good catch! You may want to add a comment:

```
<!-- we're linking the README.md files as index.html to work around https://github.com/rust-lang/mdBook/issues/984 -->
```

Reply:
Yes, but...

The checkin actions run `markdown-link-check` which grumbles about this - there are no files named _index.html_ in the source tree.

I was about to write this up as another issue, but I'll say it here: It's intriguing that this only bites us when there's a directory in the path. Normal links to _./foo.md_ work fine.

I suspect it's something to do with `mdbook` converting a _README.md_ into _index.html_ for unclear reasons (well, I know why this could be useful - every directory since 1993 has needed an _index.html_ file - but that isn't necessarily helpful here...) I'm hopeful that we can turn this off.

Reply:
Yeah the `markdown-link-check` doesn't know about the mdbook bug but it can be disabled for these links via a comment.

> I suspect it's something to do with mdbook converting a README.md into index.html for unclear reasons 

Yes it's a bug in mdbook ... see the GitHub issue I linked in the comment.

I have opened #2990 to fix these two links, so you can drop these changes from this PR and focus on the content changes :)